policyengine_us/variables/gov/local/ca/la/dss/expectant_parent_payment/ca_la_expectant_parent_payment.py

```diff
@@ -0,0 +1,13 @@
+from policyengine_us.model_api import *
+
+
+class ca_la_expectant_parent_payment(Variable):
+    value_type = float
+    entity = Person
+    definition_period = MONTH
+    label = "Los Angeles County expectant parent payment"
+    defined_for = "ca_la_expectant_parent_payment_eligible"
+
```

Comment:
right? or [] around?
```suggestion
    adds = "gov.local.ca.la.dss.expectant_parent_payment.amount"
```

Reply:
Forgot about the [] - works here